README.md

```diff
@@ -24,13 +24,48 @@ $ git submodule update
 
 If you have more issues with submodules, please check out [TROUBLESHOOTING.md](https://github.com/ourjapanlife/findadoc-frontend/blob/main/TROUBLESHOOTING.md)
 
-## Environment Setup
+## Environment and Firebase Setup
```

Comment:
Thanks for adding docs!

Reply:
From the get go you've added killer docs @ann-kilzer . I'm just trying to catch up =).